### PR TITLE
Add a RHEL7 based container image for dd-agent

### PR DESCRIPTION
**Pull request by irabinovitch**
### What does this PR do?

Add a RHEL7 based container image for dd-agent

### Motivation

Inclusion in Red Hat's [container directory](https://access.redhat.com/containers). 

### Testing Guidelines

1) Spin up a RHEL virtual machine.  
2)  docker build . --file=./Dockerfile-rhel 
3) docker run $IMAGE_ID

### CODE REVIEW

File: Dockerfile-rhel [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    PYTHONPATH=/opt/datadog-agent/agent \
+    DD_CONF_LOG_TO_SYSLOG=no \
+    NON_LOCAL_TRAFFIC=yes \
+    DD_SUPERVISOR_DELETE_USER=yes
+
+LABEL name="datadog/docker-dd-agent:latest-rhel" \
+      vendor="Datadog" \
+      version="${AGENT_VERSION}" \
+      release="" \
+      summary="A containerized ersion ofthe Datadog Agent." \
```
Comment (hkaj) [inline Dockerfile-rhel]: ersion --> version
ofthe --> of the

File: rhel/help.1 [hunk 2]
```diff
@@ -0,0 +1 @@
+open help.p: no such file or directory
```
Comment (hkaj) [inline rhel/help.1]: ❓

File: rhel/help.md [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+
+The Datadog Agent provides real-time performance tracking and visualization of your containers, orchestrators,
+operating system and application metrics.
+
+# SUPPORT
+
+Datadog's support team is available to assist you with any Datadog Agent related issues.
+Support can be reached via [ticket](http://docs.datadoghq.com/help/), chat or other channels.
+
+Please remember to [send your logs](https://github.com/DataDog/dd-agent/wiki/Send-logs-to-support)
```
Comment (hkaj) [inline rhel/help.md]: blank page - let's use https://help.datadoghq.com/hc/en-us/articles/204991415-Send-logs-and-configs-to-Datadog-via-flare-command instead?

File: Dockerfile-rhel (hunk 1 above)
Comment (irabinovitch) [reply Dockerfile-rhel]: fixed

File: rhel/help.1 (hunk 2 above)
Comment (irabinovitch) [reply rhel/help.1]: fixed

**Comment by irabinovitch** [comment]
I think I've addressed all the comments.

**Pull request merged**